app/src/pages/inside/stepPage/modals/editDefectModals/messages.js

```diff
@@ -121,6 +121,17 @@ export const messages = defineMessages({
     id: 'MakeDecisionModal.decisionForTest',
     defaultMessage: 'Decision for the test {launchNumber}',
   },
+  test: {
+    id: 'MakeDecisionModal.test',
+    defaultMessage: 'Test {launchNumber}',
+  },
+  currentSelection: {
+    id: 'MakeDecisionModal.currentSelection',
+    defaultMessage: 'Current selection',
+  },
+  seeMore: { id: 'MakeDecisionModal.seeMore', defaultMessage: 'See more' },
```

Comment:
I suggest declaring messages above, outside component code. Also, why not create a common component for collapsing control?